docs/contributing.md

```diff
@@ -90,6 +90,11 @@ To add test cases (optional) covering the change in the code generator, please r
 
 To test the templates, please perform the following:
 - Update the Petstore sample by running the shell scripts under `bin` and `bin/openapi3` folder. For example, run `./bin/python-client-petstore.sh` and `./bin/openapi3/python-client-petstore.sh` to update the Python PetStore API client under [`samples/client/petstore/python`](https://github.com/openapitools/openapi-generator/tree/master/samples/client/petstore/python) and [`samples/openapi3/client/petstore/python`](https://github.com/openapitools/openapi-generator/tree/master/samples/openapi3/client/petstore/python). For Windows, the batch files can be found under `bin\windows` folder. (If you find that there are new files generated or unexpected changes as a result of the update, that's not unusual as the test cases are added to the OpenAPI spec from time to time. If you've questions or concerns, please open a ticket to start a discussion)
+- Run the Petstore server using
+```bash
+docker pull swaggerapi/petstore
+docker run -d -e SWAGGER_HOST=http://petstore.swagger.io  -e SWAGGER_BASE_PATH=/v2 -p 80:8080 swaggerapi/petstore
```

Comment:
@slash-arun we actually encourage people to leverage the CI to test the change.

Also one needs to update host table in order to use the local Petstore server, right?

Reply:
@wing328 Got it

The reason I prefer to run the test on my local is CI needs around 25 - 30 mins to run the tests since it runs all the tests. But when I am working on python I could just run the relevant test cases much faster and multiple times, instead of waiting so long. Hence I thought this should be included.

> Also one needs to update host table in order to use the local Petstore server, right?
No, the host table need not be updated. Actually the SWAGGER_HOST env variable I added is not needed. Pushed a new commit to remove that.